presto-docs/src/main/sphinx/optimizer/push-down.rst

```diff
@@ -0,0 +1,140 @@
+=========
+Push down
+=========
+
+Presto can push down processing queries, or segments of queries, down to the
+connected data source. This means that a specific function, or other operation,
+is passed through to the underlying database for processing.
+
+The results of this push down can include the following benefits:
+
+* improved overall query performance
+* reduced network traffic between Presto and the database
+* reduced load on the database
+
+Support for push down is specific to each connector and the relevant underlying
+database or storage system.
+
+Every Presto aggregate function is eligible for push down, as long as it
```

Comment:
This suggest pushdown is for aggregations. Should it be mentioned in subsection? Maybe there should be a dedicated section for aggregation pushdown?

Maybe we should also mention what parts of query we can pushdown? We pushdown:
* filters
* aggregations
* limit

Reply:
This sentence is removed so no need to detail more of this for now.